example_docs/docs/conf.py

```diff
@@ -101,5 +100,5 @@
 # Settings for Jupyter notebooks.
 nbsphinx_execute = "never"
 nbsphinx_thumbnails = {
-    "sphinx_guide/notebook": "_static/no_image.png",
+    "sphinx_guide/notebook": "_static/images/logo.png",
```

Comment:
Demo that this works:

![Screenshot 2023-05-11 at 12 10 49 PM](https://github.com/Qiskit/qiskit_sphinx_theme/assets/14852634/6dfe49c8-3377-4c1e-8636-f3d7b7afcd0e)

Reply:
Projects generally want to add `"**": "_static/images/logo.png"` to this section so that any page that doesn't have a specified thumbnail will fallback to this one. Maybe this can be demonstrated in the code here.

Reply:
Sure, in case people copy and paste this conf.py.